Procfile

```diff
@@ -0,0 +1,2 @@
+web: bundle exec puma -t 5:5 -p ${PORT:-3000} -e ${RACK_ENV:-development}
```

Comment:
We also have this configuration in `config/puma.rb`, using it in `bin/cf-start`.